java/src/main/java/com/genexus/util/GXFile.java

```diff
@@ -84,11 +88,19 @@ public void setSource(String FileName, boolean isLocal) {
         if (Application.getGXServices().get(GXServices.STORAGE_SERVICE) != null && !isLocal) {
         		FileSource = new GXExternalFileInfo(FileName, Application.getExternalProvider());
         } else {
+                String absoluteFileName = FileName;
         		try {
-        			URI uriFile = URI.create(FileName);
+        		    if (ModelContext.getModelContext() != null)
+                    {
+                        IHttpContext webContext = ModelContext.getModelContext().getHttpContext();
+                        if((webContext != null) && (webContext instanceof HttpContextWeb)) {
+                            absoluteFileName = ModelContext.getModelContext().getHttpContext().getDefaultPath() + File.separator + FileName;
```

Comment:
I think that this change could break things when  FileName is an URL (http://localhost/myimage.png) 
Example:

&File.Source = "http://......"

Maybe this was  not supported, I'm not really sure.